src/app/(tabs)/index.tsx

```diff
@@ -56,6 +57,20 @@ const HomePage = () => {
     });
   };
 
+  // ref
+  const bottomSheetModalRef = useRef<BottomSheetModal>(null);
+
+  // variables
+  const snapPoints = useMemo(() => ["25%", "50%"], []);
+
+  // callbacks
+  const handlePresentModalPress = useCallback(() => {
+    bottomSheetModalRef.current?.present();
+  }, []);
+  const handleSheetChanges = useCallback((index: number) => {
```

Comment:
you could add a break line between line 69 and 70

Reply:
done: https://github.com/LangaraBeetles/weaup-react-native-app/pull/21/commits/ed31826f4e8627245614557447d701215e5f4b76